packages/components/src/components/divider/divider.stories.ts

```diff
@@ -0,0 +1,155 @@
+import '../../solid-components';
+import { html } from 'lit';
+import { storybookDefaults, storybookTemplate } from '../../../scripts/storybook/helper';
+const { args, argTypes, parameters } = storybookDefaults('sd-divider');
+const { generateTemplate } = storybookTemplate('sd-divider');
+
+export default {
+  title: 'Components/sd-divider',
+  component: 'sd-divider',
+  args,
+
+  argTypes,
+
+  parameters: { ...parameters },
+  decorators: [
+    (story: () => typeof html) => html`
+      <style>
+        sd-divider[orientation='vertical'] {
+          height: 120px;
+        }
+        sd-divider[orientation='horizontal'] {
+          width: 120px;
+        }
+      </style>
+      ${story()}
+    `
+  ]
+};
+/**
+ * This shows sd-divider in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  }
+};
+
+/**
+ * Use the orientation attribute to set the axis of a divider.
+ */
+
+export const Orientation = {
+  parameters: { controls: { exclude: 'orientation' } },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: { type: 'attribute', name: 'orientation' }
+      },
+      args
+    });
+  }
+};
+
+/**
+ * Use the inverted attribute to make a divider with inverted colors.
+ */
+
+export const Inverted = {
+  parameters: { controls: { exclude: 'inverted' } },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: { type: 'attribute', name: 'inverted' }
+      },
+      args
+    });
+  }
+};
+
+/**
+ * Use a wrapper to set the width and height of the divider.
+ * For horizontal dividers, only set the `width`, and for vertical dividers, adjust the `height`.
```

Comment:
The wrapper is only needed here for the story, in the end you're influencing the size of the element itself. You could even set the size with inline styles.
```suggestion
 * You can set the component's size with `width` (horizontal) or `height` (vertical) via CSS or inline-styles.
```